instrumentation/opentelemetry-instrumentation-openai/tests/test_chat_completions.py

```diff
@@ -0,0 +1,162 @@
+import json
+
+import pytest
+
+from opentelemetry.semconv._incubating.attributes import (
+    gen_ai_attributes as GenAIAttributes,
+)
+
+
+@pytest.mark.vcr()
+def test_chat_completion(exporter, openai_client):
+    llm_model_value = "gpt-4"
+    messages_value = [
+        {"role": "user", "content": "Say this is a test three times"}
```

Comment:
nit: agree this is neat. otoh, the data is larger than necessary as streaming we can get multiple chunks from just saying this is a test. We can verify that in the recorded response I think

Reply:
agree, will change prompt to reduce data a bit